src/ui/components/NotificationType.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Fin } from "../../fin";
+import { Notification } from './notifications/Notification';
+import { ButtonNotification } from './notifications/ButtonNotification';
+import { INotificationProps } from '../models/INotificationProps';
+declare var fin: Fin;
+
+/**
+ * Determines what type of notification to display in the UI
+ */
+export class NotificationType extends React.Component<INotificationProps, {}> {
+    public render(): React.ReactNode {
+        if (this.props.meta.buttons) {
```

Comment:
I think we should enum out the different types of notifications and store them in the meta as "internal" data.  This way if we need to perform specific actions based on the type further down the pipeline it will be easier to do, without putting it through another if.